go-app/main.go

```diff
@@ -19,8 +19,9 @@ type Feedback struct {
 }
 
 var (
-	feedBack = Feedback{}
-	CfgFile  = report.Config{}
+	feedBack       = Feedback{}
+	fullLogDetails = report.FullDetails{}
+	cfg_file       = report.Config{}
```

Comment:
You've done well to limit the scope of these variables and pass them around, but this var is still referencing another global variable. So I think we have an opportunity to limit the scope even further here

Reply:
Which var are you talking about ?